unix/qstrdefsport.h

```diff
@@ -79,6 +79,7 @@ Q(listen)
 Q(accept)
 Q(recv)
 Q(recvfrom)
+Q(settimeout)
```

Comment:
@omtinez So for headers in general - it's just the usual game as far as C headers go. 

For qstrs, each unique string gets assigned a unique integer which is then used in the byte code. In C code, you'll see something like this:
https://github.com/micropython/micropython/blob/master/stmhal/uart.c#L726-L751

Each of the MP_QSTR_xxx is from an auto-generated enum. So MP_QSTR_deinit would be generated in response to Q(deinit).

From my build tree, genhdr/qstrdefs.generated.h contains the line:
```
QDEF(MP_QSTR_deinit, (const byte*)"\x9e\x8d\x06" "deinit")
```

The genhdr/qstrdefs.generated.h is included once here (to create the enumeration):
https://github.com/micropython/micropython/blob/master/py/qstr.h#L39-L44
and once here (to allocate the string and hash):
https://github.com/micropython/micropython/blob/master/py/qstr.c#L90-L100

And there are a variety of sources of the qstrs. This file covers the qstrs from the py tree:
https://github.com/micropython/micropython/blob/master/py/qstrdefs.h
and then each port provides the port specific ones:
https://github.com/micropython/micropython/blob/master/stmhal/qstrdefsport.h

I don't see any module ones right now, but I imagine that there could be module specific files ones as well.